test/browser/browser_asrouter_snippets.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const {ASRouter} =
+  ChromeUtils.import("resource://activity-stream/lib/ASRouter.jsm");
+
+test_newtab({
+  async before({pushPrefs}) {
+    let data = ASRouter.state.messages.find(m => m.id === "SIMPLE_BELOW_SEARCH_TEST_1");
+    ASRouter.messageChannel.sendAsyncMessage("ASRouter:parent-to-child", {type: "SET_MESSAGE", data});
+  },
+  test: function test_simple_below_search_snippet() {
+    // Verify the simple_below_search_snippet renders in container below searchbox
+    // and nothing is rendered in the footer.
+    let container = content.document.querySelector(".below-search-snippet");
```

Comment:
Maybe add a `await ContentTaskUtils.waitForCondition` to be sure that we wait long enough for that message to arrive to content.